src/tsolvers/lasolver/Delta.h

```diff
@@ -99,7 +100,8 @@ class Delta {
     inline friend bool operator>=(const Real & c, const Delta & a);
 
     // Arithmetic overloadings
-    inline friend Delta operator+=(Delta & a, const Delta & b);
+    Delta& operator+=(const Delta & b);
+    Delta& operator+=(Delta && b);
 
     inline friend Delta operator-=(Delta & a, const Delta & b);
```

Comment:
Should we do the above for `-=`, to turn it into a member operator and add the rvalue reference version?

Reply:
Yes, that makes sense.

Reply:
Fixed. Although it seems operator -= is not used in our codebase.